src/careamics/config/tile_information.py

```diff
@@ -73,23 +73,23 @@ def only_if_tiled(cls, v: bool, values: ValidationInfo):
     @field_validator("overlap_crop_coords", "stitch_coords")
     @classmethod
     def mandatory_if_tiled(
-        cls, v: Optional[Tuple[int, ...]], values: ValidationInfo
-    ) -> Optional[Tuple[int, ...]]:
+        cls, v: Optional[tuple[int, ...]], values: ValidationInfo
+    ) -> Optional[tuple[int, ...]]:
         """
         Check that the coordinates are not `None` if tiling is enabled.
 
         The method also return `None` if tiling is not enabled.
 
         Parameters
         ----------
-        v : Optional[Tuple[int, ...]]
+        v : tuple[int, ...] or None
```

Comment:
I think
```python
v : tuple of integers, optional
```

(This is mostly a comment on `optional` vs `or None`, see [other comment](https://github.com/CAREamics/careamics/pull/137/files#r1633192490) for tuple style.)

Reply:
I am a bit torn here, because `, optional` means that you don't need to provide a value, but `Optional` means that it can be `None`. The following code is correct:

```
def my_func(a: Optional[int]) -> int:
    if a is None:
        return 0
    return a
``` 

and in there `a` is not `optional`.


Ofc, in most cases `Optional` will have a default None value... So I guess we can differentiate the two cases as follows:

```
"""
a: int or None
    A, if None then A is interpreted as 0
"""
```

vs 


```
"""
a: int, optional
    A, default value is None.
"""
```

This is not an interesting example, but there will be cases where `None` has a meaning and is not the default value. Although we should probably avoid these cases in user relevant API...

In this case, the validator should not be used directly so I would stick to the `a: int or None`, because `None` really means that we didn't pass any tile.

Reply:
I think often the standard practice is:

```python
def func(a: Optional[int]=None):
    """
    Parameters
    ----------
    a : int, optional
        Description of parameter `a`.
    """
```
vs
```python
def func(a: int=1):
    """
    Parameters
    ----------
    a: int, default=1
        Description of parameter `a`.
    """
```

Reply:
Ok looking through the function more closely maybe this is an exceptional case. But generally I would follow what I put above.

Reply:
Works for me!